tests/unit/components/message-chat-test.js

```diff
@@ -4,12 +4,20 @@ moduleForComponent('message-chat', 'Unit | Component | message-chat', {
   unit: true
 });
 
-test('#formattedContent turns URLs into links', function(assert) {
+test('#formattedContent turns full URLs into links', function(assert) {
+  let component = this.subject({
+    message: { content: 'visit https://kosmos.org for more info' }
+  });
+
+  assert.equal(component.get('formattedContent').toString(), 'visit <a href="https://kosmos.org" class="linkified" target="_blank" rel="nofollow">https://kosmos.org</a> for more info');
+});
+
+test('#formattedContent ignores URLs without protocol', function(assert) {
```

Comment:
I realize it's a bit picky, but technically this is just a domain name or FQDN, not a "URL without protocol". ;)

https://en.wikipedia.org/wiki/Uniform_Resource_Locator#Syntax